news/2020-02-04-community-contributors-2019.md

```diff
@@ -0,0 +1,212 @@
+
+Largely unspoken and unsung by many, JBHyperion's efforts have dramatically reshaped the landscape of the osu!community in ways that we wager some will never truly be able to appreciate in full, but they are undoubtedly for the better, and it is safe to say that osu! has grown enormously from his time with us.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Mo.jpg)](https://osu.ppy.sh/users/2202163)
+
+Much like his brother in arms, **-Mo-** is another of the mighty few who dip their toes into the mire of leadership, spending hundreds of hours working with dozens of people in a concerted effort to improve the mapping and modding scene for everyone, travelling true throughout some of the team's rockiest years since their inception many, many years ago.
+
+Beyond his devotion to leadership, -Mo- has also made considerable contributions to the analytical and theoretical explorations of mapping via his commentary video series, showcasing his keen interest in delving into the depth of the scene from a variety of angles and perspectives.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Chaos.jpg)](https://osu.ppy.sh/users/2628870)
+
+An organizer of the Community Mentorship Program and a long-term liaison between the osu!team and the Global Moderation Team, **Chaos** has been a stabilizing force in the community for many years now, with his efforts at home and abroad solidifying the future of the game. Beyond just this, he has also been heading the osu! Mapping Olympiad, a premier beatmapping contest series all the while.
+
+His work in managing the upkeep of the GMT and ensuring that the newest generations of osu!'s mappers and modders are guided through a labyrinthine set of customs and traditions, and helping them integrate into the game proper is essentially futureproofing the scene for years to come, whilst his efforts as a moderator ensure that the game's chat and forums remain safe for everyone.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/yaspo.jpg)](https://osu.ppy.sh/users/4945926)
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Uberzolik.jpg)](https://osu.ppy.sh/users/1314547)
+
+Essential to the continuation of the Community Mentorship Program, **yaspo and Uberzolik** have spent thousands of hours performing key tasks to ensure things stay on track.
+
+After Halfslashed stepped down, **yaspo** picked up the slack and stepped up to lead beside Okoratu, and has been responsible for running dozens of contests and discussions, hand-picking several hundreds of mentor and mentee applications, hosting meetings and so much more.
+
+**Uberzolik** has been responsible for maintaining huge swathes of data required for the project to continue, balancing this workload whilst also being a mentor AND a Beatmap Nominator himself.
```

Comment:
I kinda wish this mentioned more than just the data stuff, since I essentially fill in the blanks for yaspo or directly help him with certain things. I mainly lead mentee checkups, which is meant to assure mentee-mentor interaction is working effectively/things are being done. Besides that, I handle mentor and mentee apps alongside yaspo (and a few others, sometimes). Dunno if it's wrong for me to bring this up on my own message for the newspost lol

Reply:
i've reworded your blurb to refer to the above